src/main/java/ch/njol/skript/doc/HTMLGenerator.java

```diff
@@ -68,6 +75,7 @@ public class HTMLGenerator {
 	private final File template;
 	private final File output;
 	private final String skeleton;
+	private Map<Class<? extends Event>, List<EventValues.EventValueInfo<?, ?>>> eventValuesCache = new HashMap<>();
```

Comment:
Should clear this cache after the templates have been generated, we don't need it sitting in here.